rust/examples/config.ini

```diff
@@ -2,6 +2,14 @@
 id="ora_opensource"
 type="mec_application"
 
+#[bootstrap]
+#host="mydomain.com"
+#port="8080"
+#path="/bootstrap"
+#role="external-app"
```

Comment:
The station id comes from the `[station]` section (`id` field)  
I agree this might not be very intuitive at first sight but it is a field that was here before so I thought it would be easier to use it instead of adding the same in this section

Reply:
It is present in the `[station]` section with the `id`  
I agree that it might not be very intuitive at first sight, but this is a field that was already present and used so I thought I would be more relevant to use it instead of duplicating it in the `[bootstrap]` section

Reply:
But in your code there is in bootstrap_configuration.rs
station_id: get_mandatory_field::<String>("station_id", section)?,

And so for bootstrapping, it is the station_id field in the [bootstrap] section that is taken (and that is mandatory) and not the id field of the [station] section, no?

Reply:
Indeed, my bad, it takes is from the `[bootstrap]` section, as indicated in the configuration example docstring
I remember having an issue about it, I'll fix it and push a new version